### PR TITLE
Add `DatastoreEmulatorContainer` to gcloud module

### PR DESCRIPTION
**Pull request by oskarwidmark**
- Compatible with @google-cloud/datastore

### CODE REVIEW

**Review by eddumelendez — COMMENTED** [review]
Hi @oskarwidmark, thanks for your contribution! Do you mind updating [the documentation](https://github.com/testcontainers/testcontainers-node/blob/main/docs/modules/gcloud.md), please?

**Comment by oskarwidmark** [comment]
Hi, sorry for missing that, docs updated 👍

**Review by eddumelendez — APPROVED** [review]

**Comment by oskarwidmark** [comment]
Friendly ping @cristianrgreco 🙏

File: packages/modules/gcloud/src/datastore-emulator-container.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer, Wait } from "testcontainers";
+
+const EMULATOR_PORT = 8080;
+const CMD = `gcloud beta emulators firestore start --host-port 0.0.0.0:${EMULATOR_PORT} --database-mode=datastore-mode`;
+const DEFAULT_IMAGE = "gcr.io/google.com/cloudsdktool/cloud-sdk";
+
+export class DatastoreEmulatorContainer extends GenericContainer {
```
Comment (cristianrgreco) [inline packages/modules/gcloud/src/datastore-emulator-container.ts:7]: You'll need to export these 2 classes in the `index.ts` of the gcloud module

**Comment by cristianrgreco** [comment]
Looks like the build is failing due to a MSSQL container issue which just started happening. Opened https://github.com/testcontainers/testcontainers-node/pull/801 to verify

**Pull request merged**